### PR TITLE
Generic view header

### PR DESCRIPTION
**Pull request by edewit**
Can be used as a header for a from or view as well as a specific item with actions

### CODE REVIEW

File: src/realm/messages.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "enabled":"Enabled",
     "create":"Create",
     "createRealm": "Create realm",
+    "realmExplain": "A realm is something",
```
Comment (ssilvert) [inline src/realm/messages.json:8]: You can use this text: `A realm manages a set of users, credentials, roles, and groups. A user belongs to and logs into a realm. Realms are isolated from one another and can only manage and authenticate the users that they control.`

**Review by ssilvert — CHANGES_REQUESTED** [review]

File: src/clients/ClientsSection.tsx [hunk 2]
```diff
@@ -28,40 +29,46 @@ export const ClientsSection = () => {
...
+      <PageSection variant="light">
+        <DataLoader loader={loader}>
+          {(clients) => (
+            <TableToolbar
+              count={clients!.length}
+              first={first}
+              max={max}
+              onNextClick={setFirst}
+              onPreviousClick={setFirst}
+              onPerPageSelect={(f, m) => {
```
Comment (dlabaj) [inline src/clients/ClientsSection.tsx:46]: Change to something other then f and m

**Review by dlabaj — COMMENTED** [review]
One minor comment otherwise looks great.

**Review by dlabaj — APPROVED** [review]
LGTM

**Review by ssilvert — APPROVED** [review]

**Pull request merged**